pkg/osquery/table/launcher_info.go

```diff
@@ -60,13 +62,23 @@ func generateLauncherInfoTable(store types.GetterSetter) table.GenerateFunc {
 			return nil, err
 		}
 
-		publicKey, fingerprint, err := osquery.PublicRSAKeyFromDB(store)
+		publicKey, fingerprint, err := osquery.PublicRSAKeyFromDB(configStore)
 		if err != nil {
 			// No logger here, so we can't easily log. Move on with blank values
 			publicKey = ""
 			fingerprint = ""
 		}
 
+		uptimeBytes, _ := upTimeHistoryStore.Get([]byte("process_start_time"))
```

Comment:
should probably check the error here before processing `uptimeBytes`

Reply:
Done. I avoided returning the err because I don't want uptime blocking the rest of the info on the table, so I set a default value to use if the uptime err are not nil.